tests/chains/Cosmos/THORChain/SwapTests.cpp

```diff
@@ -761,6 +763,60 @@ TEST(THORChainSwap, SwapBnbRune) {
     // https://viewblock.io/thorchain/tx/D582E1473FE229F02F162055833C64F49FB4FF515989A4785ED7898560A448FC
 }
 
+TEST(THORChainSwap, SwapBtcCacao) {
+    Proto::Asset fromAsset;
+    fromAsset.set_chain(static_cast<Proto::Chain>(Chain::BTC));
+    Proto::Asset toAsset;
+    toAsset.set_chain(static_cast<Proto::Chain>(Chain::MAYA));
+    toAsset.set_symbol("CACAO");
+    auto&& [out, errorCode, error] = SwapBuilder::builder()
+                                         .from(fromAsset)
+                                         .to(toAsset)
+                                         .fromAddress(Address1Btc)
+                                         .toAddress(Address1Maya)
+                                         .vault(VaultBtc)
```

Comment:
How is that possible that `vault` address is the same as for `THORSwap`?

Reply:
For test purpose I don't think it matters if they have the same address